code/modules/clothing/suits/marine_armor.dm

```diff
@@ -411,6 +412,7 @@ var/list/squad_colors_chat = list(rgb(230,125,125), rgb(255,230,80), rgb(255,150
 	uniform_restricted = list(/obj/item/clothing/under/marine/officer, /obj/item/clothing/under/rank/ro_suit)
 	flags_atom = NO_SNOW_TYPE
 	flags_inventory = SMARTGUN_HARNESS
+	armor_melee = CLOTHING_ARMOR_MEDIUMHIGH //added because of the base MP armor nerf
```

Comment:
Drop these changes , it can inherit of M3 as it did. Also no need to comment in the reason, leads to bloat, just a fyi.